### PR TITLE
fix: re-create project

### PR DESCRIPTION
**Pull request by marcelosj3**

### CODE REVIEW

File: sos_brazil/asgi.py [hunk 1]
```diff
@@ -11,6 +11,6 @@
 
 from django.core.asgi import get_asgi_application
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "sos_brazil.settings")
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'sos_brazil.settings')
```
Comment (gabrielricardo7) [inline sos_brazil/asgi.py:14]: Missing black formatter…

**Review by gabrielricardo7 — CHANGES_REQUESTED** [review]

File: manage.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 def main():
     """Run administrative tasks."""
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "sos_brazil.settings")
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'sos_brazil.settings')
```
[the remark this answers is not in the document]
Comment (marcelosj3) [reply manage.py]: done

File: sos_brazil/asgi.py (hunk 1 above)
Comment (marcelosj3) [reply sos_brazil/asgi.py]: done

**Review by gabrielricardo7 — APPROVED** [review]
Ok

**Pull request merged**